infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -161,6 +161,17 @@ public SequencerServer(ServerContext serverContext) {
         setUpTimerNameCache();
     }
 
+    @Override
+    void processRequest(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r) {
+        executor.submit(() -> getHandler().handle(msg, ctx, r));
```

Comment:
We should move the naming of the threads to common configuration as well. It is just easier from debugging point of view and production support can easily make sense as well.